preonboarding/Hoya324/Sol4.java

```diff
@@ -0,0 +1,40 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Sol4 {
+    private static final int asciiCodeOfA = 65;
+    private static final int asciiCodeOfZ = 90;
+    private static final int asciiCodeOfa = 97;
+    private static final int asciiCodeOfz = 122;
+
+    private StringBuilder solution(String word) {
+        int currentCharToAscii;
+        StringBuilder stringBuilder = new StringBuilder();
+        int wordLength = word.length();
+
+        for (int i = 0; i < wordLength; i++) {
+            currentCharToAscii = word.charAt(i);
+            boolean isUppercase = currentCharToAscii >= asciiCodeOfA && currentCharToAscii <= asciiCodeOfZ;
+            boolean isLowercase =  currentCharToAscii >= asciiCodeOfa && currentCharToAscii <= asciiCodeOfz;
+            if (isUppercase) {
+                stringBuilder.append((char) (asciiCodeOfZ - (currentCharToAscii - asciiCodeOfA)));
+            }
+            else if (isLowercase) {
+                stringBuilder.append((char) (asciiCodeOfz-(currentCharToAscii - asciiCodeOfa)));
+            }
+            else {
+                stringBuilder.append((char) currentCharToAscii);
+            }
+        }
+        return stringBuilder;
```

Comment:
`Builder`는 어떤 걸 만드는 역할이라 결국에 뭔가를 만들어야 함. 
```suggestion
        return stringBuilder.toString();
```

Reply:
optional) `빌더 패턴`도 공부해보기!